Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/CompositeSequences/EconomizerMultiZone.mo

```diff
@@ -0,0 +1,185 @@
+    annotation (Line(points={{-59,10},{-28,10},{-28,8},{-28,-42},{-1,-42}},
+                                                                         color={0,0,127}));
+  connect(ecoDamLim.yRetDamPhyPosMax, econEnableDisableMultiZone.uRetDamPhyPosMax)
+    annotation (Line(points={{-59,6},{-32,6},{-32,2},{-32,-38},{-1,-38}},color={0,0,127}));
+  connect(ecoDamLim.yRetDamPosMax, econEnableDisableMultiZone.uRetDamPosMax)
+    annotation (Line(points={{-59,8},{-30,8},{-30,-40},{-1,-40}},color={0,0,127}));
+  connect(ecoMod.yRetDamPos, yRetDamPos)
+    annotation (Line(points={{81,12},{100,12},{100,40},{130,40}},color={0,0,127}));
+  connect(ecoMod.yOutDamPos, yOutDamPos)
+    annotation (Line(points={{81,8},{100,8},{100,-40},{130,-40}},color={0,0,127}));
+  connect(econEnableDisableMultiZone.yOutDamPosMax, ecoMod.uOutDamPosMax)
+    annotation (Line(points={{22,-25.2},{50,-25.2},{50,-10},{50,11},{59,11}},
+    color={0,0,127}));
+  connect(econEnableDisableMultiZone.yRetDamPosMax, ecoMod.uRetDamPosMax)
+    annotation (Line(points={{22,-32},{52,-32},{52,4},{59,4}},color={0,0,127}));
+  connect(ecoDamLim.yOutDamPosMin, ecoMod.uOutDamPosMin)
+    annotation (Line(points={{-59,15},{-20,15},{20,15},{20,12},{20,8},{59,8}},
+      color={0,0,127}));
+  connect(TCooSet, ecoMod.TCooSet) annotation (Line(points={{-130,40},{52,40},{52,19},{59,19}},
+      color={0,0,127}));
+  connect(TSup, ecoMod.TSup) annotation (Line(points={{-130,60},{50,60},{50,16},{59,16}},color={0,0,127}));
+  connect(yOutDamPos, yOutDamPos) annotation (Line(points={{130,-40},{130,-40}}, color={0,0,127}));
+  connect(yRetDamPos, yRetDamPos) annotation (Line(points={{130,40},{130,40}}, color={0,0,127}));
+  connect(econEnableDisableMultiZone.yRetDamPosMin, ecoMod.uRetDamPosMin)
+    annotation (Line(points={{22,-38},{54,-38},{54,0},{54,1},{59,1}}, color={0,0,127}));
+  annotation (defaultComponentName = "multiZoneEconomizer",
+        Icon(graphics={Rectangle(
+        extent={{-100,-100},{100,100}},
+        lineColor={0,0,127},
+        fillColor={255,255,255},
+        fillPattern=FillPattern.Solid),
+        Line(points={{-66,-36},{-42,-36},{-4,40},{34,40}}, color={28,108,200},
+          thickness=0.5),
+        Line(
+          points={{-64,40},{-38,40},{2,-40},{66,-40}},
+          color={28,108,200},
+          pattern=LinePattern.Dash,
+          thickness=0.5),
+        Line(
+          points={{34,40},{34,-36},{34,-36},{66,-36}},
+          color={28,108,200},
+          thickness=0.5)}),
+        Diagram(coordinateSystem(preserveAspectRatio=false, extent={{-120,-140},{120,140}})),
+  Documentation(info="<html>
+  <p>
+  This is multiple zone VAV AHU economizer control sequence. It calculates
+  outdoor and return air damper positions based on ASHRAE
+  Guidline 36, sections: PART5 N.2.c, N.5, N.6.c, N.7, A.17, N.12.
+  The sequence comprises the following atomic sequences:
+  <code>EconDamperPositionLimitsMultiZone</code>,
```

Comment:
You need to use the full Modelica name with a hyperlink, such as <a href=\"modelica://Buildings. ....\">Buildings. ...</a>
Otherwise, if we rename models with the rename script, these names will not get updated, and there is no mechanism to test whether all names are valid (for href, we can check that the name is correct when generating the html)